presto-pinot-toolkit/src/main/java/com/facebook/presto/pinot/query/PinotQueryGeneratorContext.java

```diff
@@ -430,26 +538,28 @@ else if (!groupByColumns.isEmpty()) {
 
     public PinotQueryGeneratorContext withOutputColumns(List<VariableReferenceExpression> outputColumns)
     {
-        LinkedHashMap<VariableReferenceExpression, Selection> newSelections = new LinkedHashMap<>();
-        outputColumns.forEach(o -> newSelections.put(o, requireNonNull(selections.get(o), String.format("Cannot find the selection %s in the original context %s", o, this))));
```

Comment:
This check on line 434 (Cannot find the selection ...) has been dropped. Was it intentional ?

Reply:
added back.